src/pages/uxp-api/reference-js/Modules/uxp/XMP/index.md

```diff
@@ -0,0 +1,116 @@
+# XMP
+**Since**: v7.2.0
```

Comment:
@kareemmyk  I have deleted the Getting started folder and moved the file here. Please make similar changes to torq-native.